tools/checks/style/hn_check/src/hn_check.py

```diff
@@ -247,4 +258,4 @@ def main():  # pragma: no cover
 
 
 if __name__ == "__main__":  # pragma: no cover
-    main()
+    main()
```

Comment:
Minor thing, missing newline here.

Reply:
good catch.  I got confused by the whitespace checker output.